server/src/main/java/com/paypal/selion/grid/servlets/transfer/Criteria.java

```diff
@@ -34,14 +34,6 @@
     String getArtifactName();
 
     /**
-     * Returns the user Id this artifact is tied to. This is essentially the folder by the name of the user that houses
-     * the artifact.
-     * 
-     * @return User Id of the criteria.
-     */
-    String getUserId();
```

Comment:
@shelar After further thought, I think we should revert this change. A `userId` is a basic condition that a `ManagedArtifact` must conform to. It allows for the server repository to be multi-tenant.